includes/admin/class-wc-rest-payments-orders-controller.php

```diff
@@ -178,6 +178,19 @@ public function capture_terminal_payment( $request ) {
 			// Actualize order status.
 			$this->order_service->mark_terminal_payment_completed( $order, $intent_id, $intent_status, $charge_id );
 
+			// TODO check if this is the right place for this functionality.
+			// Send email receipt to the customer.
+			$merchant_settings = [
+				'business_name' => WC_Payments::get_gateway()->get_option( 'account_business_name' ),
+				'support_info'  => [
+					'address' => WC_Payments::get_gateway()->get_option( 'account_business_support_address' ),
+					'phone'   => WC_Payments::get_gateway()->get_option( 'account_business_support_phone' ),
+					'email'   => WC_Payments::get_gateway()->get_option( 'account_business_support_email' ),
+				],
+			];
+			$charge            = $this->api_client->get_charge( $charge_id );
+			do_action( 'woocommerce_payments_new_receipt', $order_id, $merchant_settings, $charge );
```

Comment:
The name of this action sounds a bit generic. Perhaps a more specific name would offer more context? Something like `woocommerce_payments_terminal_email_receipt`?

Reply:
Or `woocommerce_payments_email_ipp_receipt`

Reply:
Done